#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_default_config.dpatch" by David Gil <dgil@telefonica.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debian modifications to the configuration file.

@DPATCH@
--- base-1.3.5.orig/base_conf.php.dist	2007-02-20 01:48:24.000000000 +0000
+++ base-1.3.5/base_conf.php.dist	2007-03-04 12:30:30.000000000 +0000
@@ -45,7 +45,7 @@
  But also put the preceding slash. e.g. Your URL is http://127.0.0.1/base
  set this to /base
  */
-$BASE_urlpath = '';
+$BASE_urlpath = '/acidbase';
 
 /* Unique BASE ID.  The below variable, if set, will append its value to the
  * title bar of the browser.  This is for people who manage multiple installs
@@ -67,7 +67,7 @@
  *        $foo = 'c:\tmp'    [OK]
  *        $foo = 'c:\tmp\'   [WRONG]
  */
-$DBlib_path = '';
+$DBlib_path = '/usr/share/php/adodb';
 
 
 /* The type of underlying alert database
@@ -77,7 +77,6 @@
  *  MS SQL Server : 'mssql'
  *  Oracle      : 'oci8'
  */
-$DBtype = 'mysql';
 
 /* Alert DB connection parameters
  *   - $alert_dbname   : MySQL database name of Snort alert DB
@@ -89,11 +88,15 @@
  *  This information can be gleaned from the Snort database
  *  output plugin configuration.
  */
-$alert_dbname   = 'snort_log';
-$alert_host     = 'localhost';
-$alert_port     = '';
-$alert_user     = 'snort';
-$alert_password = 'mypassword';
+
+###### Begin of variables configured through dbconfig-common
+
+require('/etc/acidbase/database.php');
+# Adjust dbconfig-common names
+if ($DBtype =='pgsql') 
+    $DBtype = 'postgres';
+
+###### End of variables configured through dbconfig-common
 
 /* Archive DB connection parameters */
 $archive_exists   = 0; # Set this to 1 if you have an archive DB
@@ -373,7 +376,7 @@
 /*
  The below line should not be changed!
  */
-$BASE_path = dirname(__FILE__);
+$BASE_path = '/usr/share/acidbase';
 
 // _BASE_INC is a variable set to prevent direct access to certain include files....
 define( "_BASE_INC", 1 );
